packages/frontend/web/components/SeriesSectionLink.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { pillarMap, pillarPalette } from '@frontend/lib/pillars';
+import { until, wide, leftCol } from '@guardian/pasteup/breakpoints';
+import { headline } from '@guardian/pasteup/typography';
+
+const leftColWidth = css`
+    ${leftCol} {
+        width: 140px;
+    }
+
+    ${wide} {
+        width: 220px;
+    }
+`;
+
+const section = css`
```

Comment:
Looking at this more closely - we should keep section and the wrapper in the `articleBody.tsx` so moving this css and `<div className={section}>` out to there, because even if we have no tag to show (edge case) we still want to have the layout otherwise the heading collapses